src/main/java/com/internet/shop/controllers/InjectDataController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class InjectDataController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        User bob = new User("bob");
+        User alisa = new User("alisa");
+        userService.create(bob);
+        userService.create(alisa);
+
```

Comment:
last 4 lines with Bob and Alisa - you can delete

Reply:
It was convinient to add users to DB by one click to check adding progucts to cart. So I have left it.